extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/StoreType.java

```diff
@@ -1,17 +1,24 @@
 package io.quarkus.quartz.runtime;
 
-import org.quartz.impl.jdbcjobstore.JobStoreTX;
-import org.quartz.simpl.RAMJobStore;
-
 public enum StoreType {
-    RAM(RAMJobStore.class.getName(), RAMJobStore.class.getSimpleName()),
-    DB(JobStoreTX.class.getName(), JobStoreTX.class.getSimpleName());
+    RAM("org.quartz.simpl.RAMJobStore", "RAMJobStore"),
+    DB("org.quartz.impl.jdbcjobstore.JobStoreCMT", "JobStoreCMT"),
```

Comment:
I would deprecate this value and add a comment that it's retained for compatibility reasons and will be removed in 1.6 (or just "in future versions").

Reply:
Great, I was not sure how to go about this. Thanks clarifying it. I'll do it and go ahead and upgrade the quickstarts too.

Reply:
Something like this should be OK:
https://github.com/quarkusio/quarkus/blob/master/independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeploymentValidator.java#L25-L36

Reply:
Done.